schedules/update_schedule.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+from datetime import timedelta
+
+from temporalio.client import Client, ScheduleUpdate, ScheduleUpdateInput
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+
+    async def update_schedule_simple(
+        input: ScheduleUpdateInput, timeout_minutes: int = 7
+    ) -> ScheduleUpdate:
+        schedule_action = input.description.schedule.action
+        schedule_action.task_timeout = timedelta(minutes=timeout_minutes)
```

Comment:
We should never really show task timeout update in a sample. We discourage altering this value.

Reply:
Yes, you want to make sure that all of the lints pass.

In this case, the `action` is of that type from MyPy's POV because it's the base class `ScheduleAction`. Right now we only have `ScheduleActionStartWorkflow` but we'll have more in the future. You'll need to use something like `if isinstance(schedule_action, ScheduleActionStartWorkflow):`